tests/e2e/logging/test_openshift-logging.py

```diff
@@ -0,0 +1,148 @@
+"""
+This file contains the testcases for openshift-logging
+"""
+
+import pytest
+import logging
+
+from tests import helpers
+from ocs_ci.ocs.resources.pod import get_all_pods, get_pod_obj
+from ocs_ci.ocs import constants
+from ocs_ci.utility import deployment_openshift_logging as obj
+from ocs_ci.utility.uninstall_openshift_logging import uninstall_cluster_logging
+from ocs_ci.framework.testlib import E2ETest, tier1
+from ocs_ci.utility.retry import retry
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+
+    * The setup will deploy openshift-logging in the cluster
+
+    * The teardown will uninstall cluster-logging from the cluster
+
+    """
+
+    def finalizer():
+        teardown(sc_obj, cbp_obj)
+
+    request.addfinalizer(finalizer)
+
+    # Deploys elastic-search operator on the project openshift-operators-redhat
+    obj.create_namespace(yaml_file=constants.EO_NAMESPACE_YAML)
```

Comment:
where do you define 'obj'?

Reply:
obj is defined in the import itself.

Reply:
ohh now I see. Thanks
Any chance you change its name to something like dep_ocp_logging_obj?

Reply:
ack

Reply:
done